float64[8] dqrd
float64[8] dqad
float64 abs_error
float64 rel_error
float64 e_n
float64 angle
float64 dot_s
float64[14] dq
